package.json

```diff
@@ -59,14 +59,14 @@
     "react-test-renderer": "^16.6.3",
     "regenerator-runtime": "^0.12.1",
     "request": "^2.83.0",
-    "rollup": "^0.66.2",
-    "rollup-plugin-alias": "^1.3.0",
-    "rollup-plugin-babel": "^4.0.3",
-    "rollup-plugin-commonjs": "^9.0.0",
-    "rollup-plugin-filesize": "^5.0.0",
-    "rollup-plugin-node-resolve": "^3.0.0",
+    "rollup": "^1.1.2",
+    "rollup-plugin-alias": "^1.5.1",
+    "rollup-plugin-babel": "^4.3.2",
+    "rollup-plugin-commonjs": "^9.2.0",
+    "rollup-plugin-filesize": "^6.0.1",
+    "rollup-plugin-node-resolve": "^4.0.0",
     "rollup-plugin-replace": "^2.1.0",
-    "rollup-plugin-uglify": "^6.0.0",
+    "rollup-plugin-uglify": "^6.0.2",
```

Comment:
Perhaps you can replace uglify with actually maintained rollup-plugin-terser? Lite package is using that as well.

Reply:
Migrated to microbundle, so that abstracts all that away, significantly simplifying the setup :).